### PR TITLE
Rename cli to bor in cmd

### PR DESCRIPTION
**Pull request by manav2401**

### CODE REVIEW

**Comment by ferranbt** [comment]
Now that I think about this, `make all` builds and creates a binary of every folder inside `cmd` to `build` so it would create two binaries one for `geth` (old cli) and one for `bor` (new cli). But, if we run `make bor` it creates `geth` binary and copies it to a `bor` binary so this might create problems with the old cli.

**Comment by manav2401** [comment]
Yeah right. We can merge this once we fully stop old cli and release the new one.

**Review by vcastellm — APPROVED** [review]

**Comment by ferranbt** [comment]
I am moving this to draft until we are clear to merge.

**Pull request closed without merging**
